app/views/placements/schools/placements/show.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= content_for :page_title, sanitize(@placement.subject_names) %>
+
+<%= render "placements/schools/primary_navigation", school: @school, current_navigation: :placements %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: placements_school_placements_path(school_id: @school.id)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-l">
+        <%= @placement.subject_names %>
+      </h2>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% if !Subject.subject_areas.values.include?(@school.phase&.downcase) %>
```

Comment:
I wonder if this logic deserves to be pulled out of this file and put somewhere else – maybe a helper method or in the decorator? 🤔 It seems like it'll be useful for the 'add' journey too. Could always pull it out later if you'd prefer.